scene/3d/camera.cpp

```diff
@@ -170,6 +181,27 @@ void Camera::set_perspective(float p_fovy_degrees, float p_z_near, float p_z_far
 	update_gizmo();
 	force_change = false;
 }
+void Camera::set_oblique(float p_fovy_degrees, Dictionary p_oblique_data, float p_z_near, float p_z_far) {
```

Comment:
It seems that `p_oblique_data` is read-only here. So `const Dictionary &` might be better.

Reply:
omw